third_party/util/atomic_writer.go

```diff
@@ -95,14 +96,14 @@ const (
 //      data to determine if an update is required.
 //  5.  A new timestamped dir is created
 //  6.  The payload is written to the new timestamped directory
-//  7.  Symlinks and directory for new user-visible files are created (if needed).
+//  7.  Symlinks and directory for new user-group-visible files are created (if needed).
```

Comment:
don't think this is an intentional change?

Reply:
It was, but can change it back. The intention was that we _used_ to only have user-visible files in the sense that it only supported Chowning the user. We now support _user and group_.